docs/pages/_app.js

```diff
@@ -22,6 +22,10 @@ import createEmotionCache from 'docs/src/createEmotionCache';
 import findActivePage from 'docs/src/modules/utils/findActivePage';
 import useRouterExtra from 'docs/src/modules/utils/useRouterExtra';
 import { LicenseInfo } from '@mui/x-data-grid-pro';
+import materialPkgJson from '../../packages/mui-material/package.json';
+import joyPkgJson from '../../packages/mui-joy/package.json';
+import systemPkgJson from '../../packages/mui-system/package.json';
+import basePkgJson from '../../packages/mui-base/package.json';
```

Comment:
We could do this now:

```suggestion
import materialPkgJson from 'packages/mui-material/package.json';
import joyPkgJson from 'packages/mui-joy/package.json';
import systemPkgJson from 'packages/mui-system/package.json';
import basePkgJson from 'packages/mui-base/package.json';
```

Previously, we couldn't because `docs/src/modules/components/AppNavDrawer.js` was imported by the other repositories, and they didn't know how to resolve the absolute import.